File: PHP/Cap04/c04ex03.php
<?php
// c04ex03.php

printf("CALCULO DE RESULTADO\n\n");

printf("Entre o 1o. valor numerico inteiro: ");
$A = intval(trim(fgets(STDIN)));

printf("Entre o 2o. valor numerico inteiro: ");
$B = intval(trim(fgets(STDIN)));

$X = $A + $B;

if ($X >= 10)
  $R = $X + 5;
else
  $R = $X - 7;

printf("Resultado = %d\n", $R);

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDIN));

?>

File: GO/Cap04/c04ex03.go
// c04ex03.go

package main
import "fmt"

func main() {
  var A, B, X, R int

  fmt.Print("Entre o 1o. valor numerico: ")
  fmt.Scan(&A)

  fmt.Print("Entre o 2o. valor numerico: ")
  fmt.Scan(&B)

  X = A + B

  if (X >= 10) {
    R = X + 5
  } else {
    R = X - 7
  }

  fmt.Println("Resultado =", R)

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}
